unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/PrivateChatWindow/IPrivateChatComponentView.cs

```diff
@@ -8,16 +9,20 @@ public interface IPrivateChatComponentView
     event Action OnClose;
     event Action<string> OnUnfriend;
     event Action<bool> OnFocused;
-    
+    event Action OnScrollUpToTheTop;
```

Comment:
i think its better to name it more like: `OnRequireMoreMessages` instead of mentioning the scroll, since you could change the implementation for a button and the interface would be the same